--- js/main.js
+++ js/main.js
@@ -1900,10 +1900,10 @@
     const r = BLOCKS[o];
     if (!r) return;
-    const l = r.pickaxe ? createPickaxeMesh(o) : r.bone ? createBoneMesh() : new THREE.Mesh(
+    const l = r.pickaxe ? createPickaxeMesh(o) : (o === 179) ? createLaserGunMesh(179) : r.bone ? createBoneMesh() : new THREE.Mesh(
         new THREE.SphereGeometry(.25, 16, 16),
         new THREE.MeshStandardMaterial({
             color: r.color,
             emissive: r.color,
             emissiveIntensity: .5
         }));
